### ISSUE TITLE
Degrade gracefully when services could not be loaded

### ISSUE THREAD

**Issue by sekruse** [yfiton/yfiton#3]
It appears that OpenJDKs do not ship with JavaFX. In those setups, the `DesktopNotifier` cannot be loaded (unless you take care of putting `jfxrt.jar` on the classpath yourself, of course):
```
Exception in thread "main" java.util.ServiceConfigurationError: com.yfiton.api.Notifier: Provider com.yfiton.notifiers.desktop.DesktopNotifier could not be instantiated
	at java.util.ServiceLoader.fail(ServiceLoader.java:232)
	at java.util.ServiceLoader.access$100(ServiceLoader.java:185)
	at java.util.ServiceLoader$LazyIterator.nextService(ServiceLoader.java:384)
	at java.util.ServiceLoader$LazyIterator.next(ServiceLoader.java:404)
	at java.util.ServiceLoader$1.next(ServiceLoader.java:480)
	at java.util.Iterator.forEachRemaining(Iterator.java:116)
	at com.yfiton.core.NotifierRegistry.reload(NotifierRegistry.java:67)
	at com.yfiton.core.NotifierRegistry.<init>(NotifierRegistry.java:45)
	at com.yfiton.core.NotifierRegistry$LazyHolder.<clinit>(NotifierRegistry.java:37)
	at com.yfiton.core.NotifierRegistry.getInstance(NotifierRegistry.java:72)
	at com.yfiton.core.YfitonBuilder.resolve(YfitonBuilder.java:64)
	at com.yfiton.core.YfitonBuilder.build(YfitonBuilder.java:77)
	at com.yfiton.cli.YfitonCli.main(YfitonCli.java:82)
Caused by: java.lang.NoClassDefFoundError: javafx/geometry/Pos
	at com.yfiton.notifiers.desktop.DesktopNotifier.<init>(DesktopNotifier.java:47)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at java.lang.Class.newInstance(Class.java:442)
	at java.util.ServiceLoader$LazyIterator.nextService(ServiceLoader.java:380)
	... 10 more
Caused by: java.lang.ClassNotFoundException: javafx.geometry.Pos
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 17 more
```

However, it seems that the `DesktopNotifier` is loaded even when it is not required (in my case, because I want to use the `SlackNotifier`). Therefore, I propose to delay crashing the app to the point where a notifier that is needed could not be loaded. This issue has been encounted on the release `1.0.0-alpha.1`.

**Comment by laurentpellegrino** [comment]
@sekruse Thanks for your report. The issue has been fixed in version 1.0.0-alpha2:

https://github.com/yfiton/yfiton/releases/tag/1.0.0-alpha.2

**Comment by sekruse** [comment]
@lpellegr Thanks a lot!
